src/components/PersonalData/PersonalData.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import TextField from "./TextField/TextField.js";
+const personalData = props => {
```

Comment:
Could you add empty like between imports and component? it would be more readable